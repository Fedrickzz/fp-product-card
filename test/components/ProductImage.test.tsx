import React from 'react';
import renderer from 'react-test-renderer';

import { ProductImage, ProductCard } from '../../src/components';
import { product2 } from '../data/products';

describe('Pruebas en ProductImage', () => {
  test('debe de mostrar el componente con la imagen pernalizada', () => {
    const wrapper = renderer.create(<ProductImage img="https://ma.png" />);

    // console.log(wrapper.toJSON());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  test('debe de mostrar el componente con el nombre del productro', () => {
    const wrapper = renderer.create(
      <ProductCard product={product2}>{() => <ProductImage />}</ProductCard>
    );

    // console.log(wrapper.toJSON());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
