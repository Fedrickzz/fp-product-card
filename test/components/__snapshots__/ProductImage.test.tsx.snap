// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en ProductImage debe de mostrar el componente con el nombre del productro 1`] = `null`;

exports[`Pruebas en ProductImage debe de mostrar el componente con la imagen pernalizada 1`] = `null`;
